packages/ui-components/src/components/DataTable/useDataTableExport.ts

```diff
@@ -19,9 +25,11 @@ export const useDataTableExport = (columns, data, title, startDate, endDate) =>
 
     // Get data from react table properties
     const header = headerGroups.map(({ headers }) =>
-      headers.map(({ Header: getHeader, id }) =>
-        typeof getHeader === 'function' ? getHeader({ column: { id } }) : getHeader,
-      ),
+      headers.map(({ Header, id }) => {
```

Comment:
I get some errors here about implicit types too. Happy for you to ts ignore them.